lua/telescope/builtin/files.lua

```diff
@@ -229,8 +230,19 @@ files.file_browser = function(opts)
     }
   end
 
+  local function get_prefix(path)
+    path = Path:new(vim.fn.fnamemodify(path, ':p'))
+
+    if path.filename:match('%.%.') then
+      path = Path:new(Path:new(path:parents()):parents())
```

Comment:
This triple path definition throws me off :laughing: (Which means i have no idea what happens here and i wrote `parents` :laughing: 
Is there an oversight in `plenary.path:parents`?

Reply:
Yeah the `plenary.path` api is a bit weird because the functions do not return paths, so we have to create new paths. It would be good if I could update it to return paths, but that might break other stuff that depends on it.

Reply:
Also I don't know why it is called `parents`. I feel like it should be called `parent` and there should be another function called `parents` or `ancestors` that will return an iterator of `parents`.

Reply:
You are right, thats because i am stupid and didn't finish reading the [pathlib spec](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.parents) (which it is based on) correctly :laughing:

Reply:
Fix in https://github.com/nvim-lua/plenary.nvim/pull/107